### ISSUE TITLE
Implement new footer design for static screens

### ISSUE THREAD

**Issue by ri0nardo** [waterthetrees/wtt_front#367]
![Image](https://user-images.githubusercontent.com/98299952/204203846-90784fe7-58c5-48d9-82b4-c5bb9e15f481.jpg)

Redesign for footer. Current footer has scrolling and clicking issues that are currently being resolved in a different ticket. See image of current footer design below.

New footer to have even spacing and be centered to the screen. See image of both desktop and mobile footer design.

To Find the design in Figma
Open the figma link
When opened, go to the left sidebar
Find pages, Desktop
When in desktop scroll down to the about screen to find the components.

![Image](https://user-images.githubusercontent.com/98299952/204203802-7f461621-a0dd-408c-a55b-e6e53bd98277.PNG)
![Image](https://user-images.githubusercontent.com/98299952/204203801-52fbd1d1-b205-4874-8ea6-64fcaed22c83.PNG)
![Image](https://user-images.githubusercontent.com/98299952/204203803-12ca36de-cbec-45ac-96ad-1d3e49360abf.PNG)

![Image](https://user-images.githubusercontent.com/98299952/204203807-a2245552-f155-4d13-91cd-0747413a1f88.jpg)

**Comment by rabbanirizwan** [comment]
@ri0nardo  kindly assign me this and also share the link of figma

**Comment by ri0nardo** [comment]
@rabbanirizwan please read the notes before starting. Figma link is attached to ticket.

**Comment by ri0nardo** [comment]
@rabbanirizwan how is progress on this ticket?

**Comment by zoobot** [comment]
@rabbanirizwan how's this going? Still working on it?

**Comment by ri0nardo** [comment]
@vernfongchao no response from assignee

**Comment by ri0nardo** [comment]
@vernfongchao can we close this ticket?

**Comment by vernfongchao** [comment]
@ri0nardo I am going to reopen this issue in the future and create a new pr fix, I forgot that I didn't add any margins to the bottom of the footer. I didn't add any margins because it was actually causing issues with the old design of other pages/added extra margins when the page wasn't filled up. Once the new designs of pages gets rolled in. ill add in a new pr to fix this and references this issue.

**Comment by ri0nardo** [comment]
@vernfongchao do you want to backlog this ticket? & just assign yourself since you seem to have a plan.

**Comment by vernfongchao** [comment]
yeah good idea! will do

**Comment by ri0nardo** [comment]
@vernfongchao im going to redesign this. Also I don't know if you added the margins, but we can ignore it for now.

**Comment by ri0nardo** [comment]
new design of footer, find it in the "static webpages" section of figma on the top left of the navigation bar. 

![Image](https://user-images.githubusercontent.com/98299952/229311112-972d6b45-6cba-4ef1-8dd4-637bd6c81d14.jpg)
![Image](https://user-images.githubusercontent.com/98299952/229311111-ab3d18b6-69dc-4df2-976d-819654c5280a.jpg)

**Comment by ri0nardo** [comment]
@vernfongchao are you still interested in the redesign of the footer? I have both the desktop and mobile designs ready. if you arent interested you can unassign yourself

**Comment by ri0nardo** [comment]
@zoobot i added a new section in the footer "Work with Us". Are you ok with what is written?

**Comment by zoobot** [comment]
> @zoobot i added a new section in the footer "Work with Us". Are you ok with what is written?

I would change "Work" to "Volunteer" since to make it explicit from the get go that this is a volunteer civic tech project.

**Comment by ri0nardo** [comment]
Updated footer to say "collaborate"!
![Footer](https://user-images.githubusercontent.com/98299952/229615340-e600f6a1-bf49-4f29-a5c2-1ada2b8f024c.jpg)
![iPhone 14 - 43](https://user-images.githubusercontent.com/98299952/229615344-818311be-18f4-46c4-95d4-811dd4bff71f.jpg)

**Comment by vernfongchao** [comment]
@ri0nardo yeah, sorry still recovering from covid, I'm actually working on this right now.

**Comment by zoobot** [comment]
Hope you feel better soon @vernfongchao Take it easy!!
